services/avs/mgmt/2020-03-20/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+pub mod models;
+pub mod operations;
+
+pub const API_VERSION: &str = "2020-03-20";
+
+pub type Error = Box<dyn std::error::Error + Send + Sync>;
+pub type Result<T> = std::result::Result<T, Error>;
+
+pub struct Configuration {
+    pub api_version: String,
+    pub client: reqwest::Client,
+    pub base_path: String,
```

Comment:
I also wonder why base_path is a field. Shouldn't this also always be "https://management.azure.com"?

Reply:
No, there are several Azure Clouds. `az cloud list` and the property is `endpoint.resourceManager`. In development, we have several Cloud instances that we work with.